src/components/Controls/ColorizationControls.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement } from "react";
+
+import {
+  DISPLAY_CATEGORICAL_PALETTE_KEYS,
+  DISPLAY_COLOR_RAMP_KEYS,
+  KNOWN_CATEGORICAL_PALETTES,
+  KNOWN_COLOR_RAMPS,
+} from "../../colorizer";
+import { useViewerStateStore } from "../../state";
+import { FlexRowAlignCenter } from "../../styles/utils";
+
+import CategoricalColorPicker from "../CategoricalColorPicker";
+import ColorRampDropdown from "../Dropdowns/ColorRampDropdown";
+import ColorRampRangeSlider from "./ColorizeControl/ColorRampRangeSlider";
+
+type ColorizationControlsProps = {
+  disabled: boolean;
+};
+
+export default function ColorizationControls(props: ColorizationControlsProps): ReactElement {
+  const dataset = useViewerStateStore((state) => state.dataset);
+  const featureKey = useViewerStateStore((state) => state.featureKey);
+
+  const categoricalPalette = useViewerStateStore((state) => state.categoricalPalette);
+  const colorRampKey = useViewerStateStore((state) => state.colorRampKey);
+  const colorRampReversed = useViewerStateStore((state) => state.isColorRampReversed);
+  const selectedPaletteKey = useViewerStateStore((state) => state.categoricalPaletteKey);
+  const setCategoricalPalette = useViewerStateStore((state) => state.setCategoricalPalette);
+  const setColorRampKey = useViewerStateStore((state) => state.setColorRampKey);
+  const setColorRampReversed = useViewerStateStore((state) => state.setColorRampReversed);
+
+  const isFeatureSelected = dataset !== null && featureKey !== null;
+  const isFeatureCategorical = isFeatureSelected && dataset.isFeatureCategorical(featureKey);
+  const featureCategories = isFeatureCategorical ? dataset.getFeatureCategories(featureKey) || [] : [];
+
+  return (
+    <>
+      {/* TODO: Once the color ramp dropdown is refactored, change gap to 22px */}
```

Comment:
Part of a final set of changes that will be layout pt. 5, where I'm going to change the styling on the color ramp dropdown to match the Figma designs. (and likely do some refactoring on selection dropdowns in the process)

<img width="673" height="293" alt="Image" src="https://github.com/user-attachments/assets/661815c9-3c84-49aa-97c7-f99c5bf01b8d" />